controllers/cluster_controller.go

```diff
@@ -400,15 +424,41 @@ func GetConfigMap(context ctx.Context, cluster *fdbtypes.FoundationDBCluster, ku
 
 	for processClass, count := range desiredCounts {
 		if count > 0 {
-			filename := fmt.Sprintf("fdbmonitor-conf-%s", processClass)
-			if connectionString == "" {
-				data[filename] = ""
-			} else {
-				conf, err := GetMonitorConf(cluster, processClass, nil, nil)
+			if processClass == fdbtypes.ProcessClassStorage {
+				storageServersPerDisk, err := convertStringSetToIntSet(cluster.Status.StorageServersPerDisk)
 				if err != nil {
 					return nil, err
 				}
-				data[filename] = conf
+				// If the status field is not initialized we fallback to only the specified count
+				// in the cluster spec. This should only happen in the initial phase of a new cluster.
+				if len(cluster.Status.StorageServersPerDisk) == 0 {
+					log.Info("GetConfigMap", "fallback to", cluster.GetStorageServersPerPod(), "storageServersPerDisk", storageServersPerDisk)
```

Comment:
If we're going to leave this logging statement in, can we add some more context?